tests/test_dt_expr.py

```diff
@@ -263,6 +263,65 @@ def test_abs_all_stypes():
 
 
 
+#-------------------------------------------------------------------------------
+# exp()
+#-------------------------------------------------------------------------------
+
+def test_exp():
+    from datatable import exp
+    import cmath
```

Comment:
```suggestion
    import math
```
because `cmath` is for complex math